changelogs/CHANGELOG.rst

```diff
@@ -5,6 +5,21 @@ Cisco Iosxr Collection Release Notes
 .. contents:: Topics
 
 
+v1.2.0
+======
+
+Major Changes
```

Comment:
```suggestion
Minor Changes
```
We decided on module addition would go under Minor and if there's breaking change that'd go under Major